ocamlformat.opam

```diff
@@ -29,6 +29,7 @@ depends: [
   "cmdliner"
   "dune" {>= "2.2.0"}
   "fix"
+  "dune-build-info" {>= "2.6.0"}
```

Comment:
The version constraint is not necessary (and the fix won't be there until 2.7.0)

Reply:
So AFAIU this should be removed and above changed to `"dune" {>= "2.7.0"}`.